.buildkite/docker/docker-compose.yaml

```diff
@@ -1,24 +1,25 @@
+# No ports are exposed by default because tests run concurrently on a single CI instance
```

Comment:
I'm almost positive this isn't actually true

Reply:
Yeah, I couldn't quickly find it explicitly stated, but this page implies build agents only ever run one thing at a time: https://buildkite.com/docs/tutorials/parallel-builds

Otherwise there would be no reason to do the "vertical scaling" they describe

Reply:
This page mentions multiple agents on the same machine which implies concurrent builds.

Reply:
If you configure them that way. We don't, and they don't by default and we use their amazon stack